ReSharper.FSharp/src/FSharp.Psi/src/Impl/Cache2/FSharpProvidedParameter.cs

```diff
@@ -34,7 +35,10 @@ myInfo switch
         _ => ParameterKind.VALUE
       };
 
-    public bool IsParameterArray => false;
+    public bool IsParameterArray =>
+      myInfo is ProxyProvidedParameterInfoWithContext x &&
+      x.GetAttributeConstructorArgs(null, "System.ParamArrayAttribute") != null;
```

Comment:
Would it be easy to add tests for cases like this?

Reply:
yes, there will be a test for that